build/mac/download_hermetic_xcode.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env vpython3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Copyright 2018 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import os
+import platform
+import plistlib
+import subprocess
+import sys
+import pkg_resources
+
+try:
+    from urllib2 import URLError
+except ImportError:  # For Py3 compatibility
+    from urllib.error import URLError # pylint: disable=no-name-in-module,import-error
+
+import deps
+from deps_config import DEPS_PACKAGES_URL
+
+def LoadPList(path):
+  """Loads Plist at |path| and returns it as a dictionary."""
+  if sys.version_info.major == 2:
+    return plistlib.readPlist(path)
+  with open(path, 'rb') as f:
+    return plistlib.load(f)
+
+# This contains binaries from Xcode 13.2.1 13C100, along with the macOS 12 SDK
+XCODE_VERSION = '13.2.1'
+HERMETIC_XCODE_BINARY = (DEPS_PACKAGES_URL +
```

Comment:
I think we need to add deps-reviewers to `script/deps.py`. The `DEPS_PACKAGES_URL` value is critical, we shouldn't allow anyone to change it easily.